src/test/java/BurgerTest.java

```diff
@@ -14,9 +15,14 @@
 
```

Comment:
@RunWith(MockitoJUnitRunner.class) - добавь аннотацию - без неё будет ошибка java.lang.NullPointerException: Cannot invoke "praktikum.Bun.getPrice()" because "this.bun" is null